.azure-pipelines/stages/_release-template-stage.yml

```diff
@@ -0,0 +1,33 @@
+# Template steps for the release pipeline
+
+parameters:
+- name: is1ES
+  displayName: 'Is 1ES pipeline'
+  type: boolean
+  default: false
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.11'
+  displayName: 'Set Up Python'
+- script: python -m pip install --upgrade pip build wheel
+  displayName: 'Install Python build dependencies'
+- script: python -m build
+  displayName: 'Build ONNX Script wheel'
+- task: CopyFiles@2
+  displayName: 'Copy Python Wheel to: $(Build.ArtifactStagingDirectory)'
+  inputs:
+    SourceFolder: 'dist'
+    Contents: '*.*'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)'
+- ${{ if eq(parameters.is1ES, false) }}:
```

Comment:
Could you add a comment on why this conditional is needed?

Reply:
This is needed to enable releas to become compliant with 1ES standards